src/seam-connect/client.ts

```diff
@@ -111,4 +159,45 @@ export class Seam extends Routes {
       throw error
     }
   }
+
+  static async getClientSessionToken(
+    ops: CSTParams
+  ): Promise<APIResponse<ClientSessionResponse>> {
+    let headers: AxiosRequestHeaders = {}
+
+    if (ops.publishableKey?.startsWith("seam_pk")) {
+      // frontend mode
+      headers["seam-publishable-key"] = ops.publishableKey
+    } else if (ops.apiKey?.startsWith("seam_")) {
+      // backend mode
+      headers["seam-api-key"] = ops.apiKey
+    }
+    headers["seam-user-identifier-key"] = ops.userIdentifierKey
+
+    try {
+      const response: AxiosResponse & {
+        data: { client_session: ClientSession }
+      } = await axios.post(
+        (ops.endpoint ?? "https://connect.getseam.com") +
```

Comment:
Can we re-use `getSeamClientOptionsWithDefaults` here? We are missing the fallback to `SEAM_API_URL`.